python/tvm/rpc/server.py

```diff
@@ -56,7 +57,7 @@ def _server_env(load_library, work_path=None):
         temp = util.tempdir()
 
     # pylint: disable=unused-variable
-    @tvm._ffi.register_func("tvm.rpc.server.workpath")
+    @tvm._ffi.register_func("tvm.rpc.server.workpath", override=True)
```

Comment:
shall we update `override=True` to `can_override=True`, since it's changing in the C++?

Reply:
In this case, we changed the c++ argument because override is a keyword in c++11, the python API change would be bigger and let us keep it as it is

Reply:
okay, just documenting that I found this kwarg confusing when first learning the code :)